src/main/scala/SIFExtendedTransformer.scala

```diff
@@ -165,6 +174,10 @@ trait SIFExtendedTransformer {
 
   val skip: Seqn = Seqn(Seq(), Seq())()
 
+  private def defaultReportError(ae: AbstractError): Unit = {
+    println(ae)
```

Comment:
Why is this a `println` statement? Plugins have an `error` variable that is filled with errors and which is used by, e.g., Gobra, to check whether a transformation was successful or not

Reply:
Oh I see! It's meant that `reportError` is set to a different one (c.f. `SIFPlugin`). I'm not sure whether I prefer `println` over a runtime exception...

Reply:
I don't have any opinion I think, I'll change it to a runtime exception.